charts/openshift-etcd-backup/templates/pv.yaml

```diff
@@ -0,0 +1,24 @@
+{{- if .Values.persistence.nfs.enabled }}
```

Comment:
Instead of creating a PV you could probably configure the NFS volume directly in `spec.volumes` of the pod.
https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.19/#nfsvolumesource-v1-core

Reply:
cleaned!